modules/mod_tags_similar/src/Helper/TagsSimilarHelper.php

```diff
@@ -194,7 +194,11 @@ public static function getList(&$params)
 			$query->order($query->rand());
 		}
 
-		$query->setLimit((int) $params->get('maximum', 5));
+		if ($params->get('maximum','5') > 0)
```

Comment:
```suggestion
		if ($params->get('maximum', '5') > 0)
```
Fix PHP CS for drone.